BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/BTListenerThread.java

```diff
@@ -8,63 +8,109 @@
 import android.bluetooth.BluetoothAdapter;
 import android.bluetooth.BluetoothServerSocket;
 import android.bluetooth.BluetoothSocket;
+import android.os.CountDownTimer;
+import android.os.Handler;
+import android.os.Message;
 import android.util.Log;
 
+import org.json.JSONObject;
+
 import java.io.IOException;
 import java.util.UUID;
 
 
-public class BTListenerThread extends Thread {
+class BTListenerThread extends Thread {
+
+    private final BTListenerThread that = this;
 
     public interface  BtListenCallback{
-        public void GotConnection(BluetoothSocket socket);
-        public void ListeningFailed(String reason);
+        void GotConnection(BluetoothSocket socket,String peerId,String peerName,String peerAddress);
+        void ListeningFailed(String reason);
     }
 
-    private BtListenCallback callback;
+    private final CountDownTimer HandShakeTimeOutTimer = new CountDownTimer(4000, 1000) {
+        public void onTick(long millisUntilFinished) {
+            // not using
+        }
+        public void onFinish() {
+            HandShakeFailed("TimeOut");
+        }
+    };
+
+    private String peerIdentifier = "";
+    private String peerName = "";
+    private String peerAddress = "";
+    private final String shakeBackBuf = "shakehand";
+
+    private BTHandShakeSocketTread mBTHandShakeSocketTread = null;
+    private String mInstanceString = "";
```

Comment:
As IntelliJ's own code validation calls out, this variable is assigned but never read. So what is its purpose?